app/src/main/java/com/eveningoutpost/dexdrip/webservices/WebServiceProfile.java

```diff
@@ -0,0 +1,20 @@
+package com.eveningoutpost.dexdrip.webservices;
+
+import org.json.JSONArray;
+
+/**
+ * @author James Woglom (j@wogloms.net)
+ */
+
+public class WebServiceProfile extends BaseWebService {
+
+    private static String TAG = "WebServiceProfile";
+
+    public WebResponse request(String query) {
```

Comment:
What does this class do? The `query` parameter is not used inside the `request` method.

Reply:
Removed this class and related profile route code.